components/appstream-calculator.tsx

```
@@ -0,0 +1,433 @@
+import { Label } from '@/components/ui/label';
+import { Card, CardContent } from '@/components/ui/card';
+import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from '@/components/ui/select';
+import { Input } from '@/components/ui/input';
+import { Slider } from '@/components/ui/slider';
+import { calculateAppStreamPricing, fetchAppStreamBundles, fetchAppStreamConfig } from '@/lib/api';
+import { regions } from '@/lib/regions';
+import { Button } from '@/components/ui/button';
+import { AppStreamUsagePattern } from '@/components/appstream-usage-pattern';
+import CostSummaryPanel from './cost-summary-panel';
+
+interface AppStreamUsagePattern {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'AppStreamUsagePattern'. Consider to delete it or rename it.

'AppStreamUsagePattern' is defined here:


(lint/suspicious/noRedeclare)